thirdeye/thirdeye-pinot/src/test/resources/org/apache/pinot/thirdeye/detection/validators/entity-pipeline-config-2.yaml

```diff
@@ -0,0 +1,68 @@
+detectionName: testPipeline
+description: My test pipeline
+type: COMPOSITE_ALERT
```

Comment:
Actually do we really need to have a "type" here?  The type could be inferred from the underline schema.

Reply:
Yes, this is not a compulsory field. If not specified, I am defaulting it to a METRIC alert.

Reply:
We can add more layers of smart features like auto resolve types, auto populate fields etc. but I'd like to keep the first phase simple.